api/external/cfgmgmt/cfgmgmt.proto

```diff
@@ -18,48 +18,90 @@ import "api/external/common/version/version.proto";
 
 import "google/protobuf/struct.proto";
 import "google/api/annotations.proto";
+import "protoc-gen-swagger/options/annotations.proto";
 
 // for option (chef.automate.api.policy)
 import "components/automate-grpc/protoc-gen-policy/api/annotations.proto";
 // for option (chef.automate.api.iam.policy)
 import "components/automate-grpc/protoc-gen-policy/iam/annotations.proto";
 
 service ConfigMgmt {
+  /*
+  GetNodes
+
+  Returns a list of infra nodes that have checked in to Automate. 
+  Adding a filter makes a list of all nodes that meet the filter criteria. 
+  Supports pagination, filtering, and sorting.
+	*/
```

Comment:
Do we want to add information about how filters with the same field are "OR"ed together and filters across different fields are "AND"ed together?

Reply:
ah ya, i can add a sentence or two about that! good idea